### PR TITLE
Add a puppet manifest for Transition RDS db_admin machines

### PR DESCRIPTION
**Pull request by issyl0**
- Relies on https://github.com/alphagov/govuk-aws/pull/271.

### CODE REVIEW

File: modules/govuk/manifests/node/s_transition_db_admin.pp [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+  # govuk_postgresql::server::db defined type
+  include ::govuk_postgresql::server::not_slave
+
+  # Ensure the client class is installed
+  class { '::govuk_postgresql::client': } ->
+
+  # include all PostgreSQL classes that create databases and users
+  class { '::govuk::apps::transition::db': } ->
+
+  $packages = [
```
Comment (surminus) [inline modules/govuk/manifests/node/s_transition_db_admin.pp]: this can be removed

File: modules/govuk/manifests/node/s_transition_db_admin.pp [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    'mysql-client-5.5',
+  ]
+
+  package { $packages:
```
Comment (surminus) [inline modules/govuk/manifests/node/s_transition_db_admin.pp]: this can be removed

File: modules/govuk/manifests/node/s_transition_db_admin.pp [hunk 3]
```diff
@@ -0,0 +1,54 @@
+# == Class: Govuk_Node::S_db_admin
```
Comment (surminus) [inline modules/govuk/manifests/node/s_transition_db_admin.pp]: incorrect class name

Comment (issyl0) [reply modules/govuk/manifests/node/s_transition_db_admin.pp]: :woman_facepalming:

**Comment by issyl0** [comment]
I fixed the linting errors.

**Comment by surminus** [comment]
Not sure why this is failing, but merge when green :)

**Comment by surminus** [comment]
Rebuilding should make the tests pass - merge away!

**Pull request merged**